Library/Homebrew/test/test_apply.rb

```diff
@@ -0,0 +1,27 @@
+require "testing_env"
+require "patch"
+
+class ApplyTests < Homebrew::TestCase
```

Comment:
Yup, and it also reduces the line count. I refreshed the PR with them incorporated into the existing files, if you'd like to have a look. Also, I made some minor tweaks to what was already, so not sure if you want that broken out in a separate commit.